test/integration_test/storkctl_migrationschedule_test.go

```diff
@@ -0,0 +1,335 @@
+	require.Equal(t, migrationSchedule.Spec.Template.Spec.IncludeNetworkPolicyWithCIDR, actualMigrationSchedule.Spec.Template.Spec.IncludeNetworkPolicyWithCIDR, "MigrationSchedule IncludeNetworkPolicyWithCIDR mismatch")
+
+	if inputSchedulePolicyInterval != "" {
+		//we will validate the created schedule policy has correct interval minutes value as well
+		expectedInterval, err := strconv.Atoi(inputSchedulePolicyInterval)
+		if err != nil {
+			return err
+		}
+		actualSchedulePolicy, err := storkops.Instance().GetSchedulePolicy(migrationScheduleName)
+		require.NoError(t, err, "Unable to get the created schedule policy")
+		require.Equal(t, expectedInterval, actualSchedulePolicy.Policy.Interval.IntervalMinutes, "MigrationSchedule SchedulePolicy interval-minutes mismatch")
+	}
+	return nil
+}
+
+func migrationScheduleCleanup(t *testing.T, migrationScheduleName string, migrationScheduleNs string) {
+	logrus.Info("Cleaning up created resources")
+	// We need to delete migration schedule and also schedule policy if created.
+	DeleteAndWaitForMigrationScheduleDeletion(t, migrationScheduleName, migrationScheduleNs)
+	DeleteAndWaitForSchedulePolicyDeletion(t, migrationScheduleName)
+}
+
+func DeleteAndWaitForMigrationScheduleDeletion(t *testing.T, name string, namespace string) {
+	err := storkops.Instance().DeleteMigrationSchedule(name, namespace)
+	if err != nil {
+		logrus.Errorf("Unable to delete migration schedule %s/%s", namespace, name)
+	}
+	f := func() (interface{}, bool, error) {
+		logrus.Infof("Checking if migration schedule resource is successfully deleted")
+		_, err := storkops.Instance().GetMigrationSchedule(name, namespace)
+		if err == nil {
+			return "", true, fmt.Errorf("get migration schedule : %s/%s should have failed", namespace, name)
+		}
+		if !errors.IsNotFound(err) {
+			logrus.Infof("unexpected err: %v when checking deleted migration schedule: %s/%s", err, namespace, name)
+			return "", true, err
+		}
+		//deletion done
+		logrus.Infof("Migration Schedule %s/%s successfully deleted", namespace, name)
+		return "", false, nil
+	}
+	_, err = task.DoRetryWithTimeout(f, defaultWaitTimeout, 2*time.Second)
+	require.NoError(t, err, "Unable to delete migration schedule %s/%s", namespace, name)
+}
+
+func createPrerequisiteResources(t *testing.T) {
+	createClusterPairs(t)
+	createNamespace(t)
+	createResourceTransformation(t, defaultNs)
+	createResourceTransformation(t, "test-ns")
+}
+
+// Create 2 clusterPairs each for async-dr and sync-dr in each of the default and admin namespaces
+// to be used by the migrationSchedules in this testSuite
+func createClusterPairs(t *testing.T) {
+	options := make(map[string]string)
+	syncClusterPairObject := generateClusterPairObject(syncDrClusterPair, defaultNs, options)
+	syncAdminClusterPairObject := generateClusterPairObject(syncDrAdminClusterPair, adminNs, options)
```

Comment:
any reason we are not creating clusterpair using storkctl?

Reply:
No particular reason. I think I had just looked at the existing tests and followed that to create cluster pairs